README.md

```diff
@@ -1,3 +1,6 @@
+[![All Contributors](https://img.shields.io/badge/all_contributors-12-orange.svg?style=flat-square)](#contributors)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
```

Comment:
I think we can link to `CONTRIBUTING.md` here. Happy to have a snippet in that file to link to `makeapullrequest` for people who would like to know how to make a pull request